extensions/import/src/wizard/pages/summaryPage.ts

```diff
@@ -178,6 +183,28 @@ export class SummaryPage extends ImportPage {
 		});
 	}
 
+	/**
+	 * Removes authentication related properties from connection string as SQL Tools Service now supports
+	 * 'EnableSqlAuthenticationProvider' which sets the below properties on connection string that conflict with access token:
+	 *   1. User Id
+	 *   2. Authentication
+	 * Since we need to set access token, we cannot use the same connection string as is.
+	 * @param connString Connection string to fix
+	 * @returns Updated connection string
+	 */
+	private updateConnectionStringForAccessToken(connString: string): string {
```

Comment:
How about something like this? Easier to add/remove properties. 
 return connString ? connString.split(';').filter(prop =>
    !['user', 'uid', 'password', 'pwd', 'authentication'].some(prefix => prop.toLocaleLowerCase().startsWith(prefix))
  ).join(';') : connString;